ui/apps/platform/src/hooks/patternfly/useSelectToggle.ts

```diff
@@ -5,6 +5,7 @@ interface SelectToggleReturn {
     isOpen: boolean;
     /** Callback that fires when the toggle element is clicked in the component */
     onToggle: (
+        // TODO: add this as a param once all Selects are using PF5
```

Comment:
FWIW I was working on a [branch](https://github.com/stackrox/stackrox/pull/10224) to unify some duplicate functionality hooks before the PF5 work (but held to not create conflicts) that might make it so that we don't need the extra param and can keep the hook component-agnostic.

Reply:
Compatible with the future contribution by Dave, the solution that I suggested is remove argument from `onToggle` function definition and calls (because its implementation ignores it).

Reply:
Oooh yes I forgot due to vacation brain! Mark's fix is better (and more correct). Removing all parameters that are ignored anyway should let us pass the function to a PF component directly in a compact way ala `onToggle={onToggle}` since the arguments will no longer be incompatible.